codecov_cli/helpers/git.py

```diff
@@ -82,3 +90,13 @@ def parse_git_service(remote_repo_url: str):
             extra=dict(remote_repo_url=remote_repo_url),
         )
         return None
+
+
+def is_fork_pr(pr_num, slug, service):
+    decoded_slug = decode_slug(slug)
+    git_service = get_git_service(service)
+    if git_service:
+        pull_dict = git_service.get_pull_request(decoded_slug, pr_num)
+        if pull_dict and pull_dict["head"]["slug"] != decoded_slug:
```

Comment:
do we need to decode the slug? if the PR API request gives us both the head and the base, we can just see if they're different?
```
if pull_dict and pull_dict["head"]["slug"] != pull_dict["base"]["slug"]:
```

Reply:
you're right we ca do it both ways, but we need to decode it since we need it in the API request to get the pull request